test/unit/content-info.test.js

```diff
@@ -23,6 +24,23 @@ describe('contentInfo module', function () {
         });
     });
 
+    // eslint-disable-next-line max-len
+    it('Should detect mp3 response stream if content-type is not present', function () {
```

Comment:
This will not work in browser tests.

```suggestion
    // eslint-disable-next-line max-len
    (typeof window !== 'undefined' ? it.skip : it)('Should detect mp3 response stream if content-type is not present', function () {
```